app/internal/translation.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional
+
+from iso639 import languages
+from sqlalchemy.exc import SQLAlchemyError
+from textblob import TextBlob, download_corpora
+from textblob.exceptions import NotTranslated
+
+from app.database.database import SessionLocal
+from app.database.models import User
+
+download_corpora.download_all()
```

Comment:
Maybe change it to a lazy call? (so we will call it in the first time it's needed and not always)

Reply:
this is the same as other nltk uses in other pull requests
also will only take time on the first time
after that is fast